<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantsPane</class>
 <widget class="QWidget" name="ParticipantsPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>105</width>
    <height>464</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="participantsLabel">
     <property name="text">
      <string>Participants</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="indentation">
      <number>15</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Participant</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Color</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Read Write</string>
      </property>
      <property name="background">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>104</red>
         <green>173</green>
         <blue>245</blue>
        </color>
       </brush>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="background">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>104</red>
         <green>173</green>
         <blue>245</blue>
        </color>
       </brush>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
      <item>
       <property name="text">
        <string>Owner</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="background">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Read Only</string>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Waiting</string>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="promotePushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Promote</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="demotePushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Demote</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="connectInfoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
